matchbot-cli.php

```diff
@@ -71,25 +71,33 @@
     new ExpireMatchFunds($psr11App->get(DonationRepository::class)),
     $psr11App->get(HandleOutOfSyncFunds::class),
     new RedistributeMatchFunds(
-        $psr11App->get(CampaignFundingRepository::class),
-        $now,
-        $psr11App->get(DonationRepository::class),
-        $psr11App->get(LoggerInterface::class),
+        campaignFundingRepository: $psr11App->get(CampaignFundingRepository::class),
+        entityManager: $psr11App->get(EntityManagerInterface::class),
+        now: $now,
+        donationRepository: $psr11App->get(DonationRepository::class),
+        logger: $psr11App->get(LoggerInterface::class),
+        bus: $psr11App->get(RoutableMessageBus::class),
     ),
```

Comment:
instead of making this and similar entries longer, it might be better to replace the whole thing and let PHP-DI instantiate the commands? It can often do it all by reflection on the constructor params:

github won't let me do a suggestion on a merged PR quite reasonably, but e.g.

```php
    $psr11App->get(RedistributeMatchFunds::class),
```

Reply:
I don't think this is possible until `$now` is replaced with something DI-able? As the changes were already getting big I was trying to avoid refactors of that sort.